src/tests/request.post.spec.js

```diff
@@ -96,3 +96,15 @@ describe('create request and new request notifications', () => {
       });
   });
 });
+
+describe('PATCH /notifications/all', () => {
+  it('should return a 201 status when all notification are marked as read', (done) => {
+    chai.request(app)
+      .patch('/api/v1/notifications/all')
+      .set('Authorization', helper.createToken(5, 'manager@gmail.com', true, 'manager'))
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
+        done();
+      });
+  });
+});
```

Comment:
can you test the negative scenario? e.g: what will happen if the token is invalid?

Reply:
But this was tested before in the same test file